File: Famoser.RememberLess.Data/Entities/NoteEntity.cs
using System;
using System.Runtime.Serialization;

namespace Famoser.RememberLess.Data.Entities
{
    [DataContract]
    public class NoteEntity
    {
        [DataMember]
        public Guid Guid;

        [DataMember]
        public string Content;

        [DataMember]
        public string Description;

        [DataMember]
        public DateTime CreateTime;

        [DataMember]
        public string IsCompleted
        {
            set
            {
                if (value == "1" || value == "true" || value == "True")
                    IsCompletedBool = true;
            }
            get { return IsCompletedBool.ToString(); }
        }

        public bool IsCompletedBool;
    }
}

File: Famoser.RememberLess.Webpage/models/entities/NoteEntity.php
<?php
/**
 * Created by PhpStorm.
 * User: Florian Moser
 * Date: 11.01.2016
 * Time: 12:42
 */

namespace famoser\rememberless\webpage\models\entities;


use famoser\rememberless\webpage\models\Note;

class NoteEntity extends Note
{
    public function __construct(Note $note)
    {
        //$this->Id = $ds->Id; Id censored
        $this->Guid = $note->Guid;
        $this->Content = $note->Content;
        $this->Description = $note->Description;
        $this->CreateTime = $note->CreateTime;
        $this->IsCompleted = $note->IsCompleted;
    }
}